ios/CBIOSession+Bridging.m

```diff
@@ -9,7 +9,7 @@ -(NSDictionary*) toDict {
         @"code": self.code ? self.code : NSNull.null,
         @"state": self.state ? self.state : NSNull.null,
         @"full_device": @(self.fullDevice),
-        @"full_device_state": [CBIORCTUtil fullDeviceState: self.fullDeviceState],
+        @"full_device_state": [CBIORCTUtil fullDeviceState: self.fullDevice],
```

Comment:
`full_device_state` should be redundant now, I removed it from the Android wrapper.